src/GraphQL.Relay.Todo/GraphQL.Relay.Todo.csproj

```diff
@@ -6,10 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="GraphQL.MicrosoftDI" Version="5.2.0" />
-    <PackageReference Include="GraphQL.SystemTextJson" Version="5.2.0" />
-    <PackageReference Include="GraphQL.Server.Transports.AspNetCore" Version="6.0.0" />
-    <PackageReference Include="GraphQL.Server.Ui.GraphiQL" Version="6.0.0" />
+    <PackageReference Include="GraphQL.MicrosoftDI" Version="5.3.1" />
+    <PackageReference Include="GraphQL.SystemTextJson" Version="5.3.1" />
```

Comment:
This should be GraphQLTestVersion, right?

Reply:
And ditto here if this is a sample project then we don’t need to use the prop

Reply:
seems so